specs/core/1_custody-game.md

```diff
@@ -393,9 +494,14 @@ def process_bit_challenge(state: BeaconState,
         assert record.challenger_index != challenge.challenger_index
         assert record.responder_index != challenge.responder_index
     # Verify the responder key
-    assert verify_custody_key(state, CustodyKeyReveal(
-        revealer_index=challenge.responder_index,
-        period=epoch_to_custody_period(slot_to_epoch(attestation.data.slot)),
+    assert verify_custody_key(state, RandaoKeyReveal(
+        revealed_index=challenge.responder_index,
+        epoch=get_randao_epoch_for_custody_period(
+            get_validators_custody_reveal_period(
+                state=state,
+                index=challenge.responder_index,
+                epoch=slot_to_epoch(attestation.data.slot)),
+            challenge.responder_index),
```

Comment:
IMO we don't even need the `verify_custody_key` function here; just use the one-liner we use in phase 0:

```python
    epoch=get_randao_epoch_for_custody_period(
            get_validators_custody_reveal_period(
                state=state,
                index=challenge.responder_index,
                epoch=slot_to_epoch(attestation.data.slot)),
            challenge.responder_index)
    assert bls_verify(responder.pubkey, hash_tree_root(epoch), challenge.responder_key, get_domain(state, DOMAIN_RANDAO, epoch))
```

This way we can remove the complexity of wrapping the custody key in a pointless reveal object with extraneous fields (the empty mask) and furthermore reduce the number of `verify_custody_key` calls to 1, which allows the function to be removed entirely (see comment above).

Reply:
Yes, done